RemoteContainers.md

```diff
@@ -0,0 +1,185 @@
+# Using m365pnp/spfx Image in Visual Studio Remote Containers
+
+Visual Studio Code Remote Containers allows you to work with SPFx solutions in containers that Visual Studio will run automatically for you so you don't need to care running `docker run`. In this it is also easier to make sure that all members of the project are using the same development environment.
+
+## Prerequisites
+
+There might be two options of using Visual Studio Code remote containers with m365pnp/spfx image:
+
+- Using locally installed Docker, on your desktop/laptop
+- Running containers in cloud using GitHub Codespaces (requires a paid GitHub plan)
+
+When you run in a remote container in GitHub Codespaces, you do NOT need to install any of the following on your laptop/desktop:
+
+- Node.js
+- yeoman
+- gulp
+- Docker
+- git
+
+What you will need locally is only Visual Studio!
+
+### Locally Installed Docker
+
+For using that option you need to have locally installed the following software:
+
+- Visual Studio Code
+- Visual Studio Code extansion, Remote - Containers https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers
```

Comment:
Let's mark the URL as a hyperlink for easier navigation

Reply:
Is it better like this?